google-cloud-spanner/pom.xml

```diff
@@ -60,7 +60,8 @@
         <executions>
           <execution>
             <goals>
-              <goal>integration-test</goal>
+              <!--<goal>integration-test</goal>-->
+              <goal>verify</goal>
```

Comment:
This was commited by mistake (I overrode it with my subsequent commit, https://github.com/GoogleCloudPlatform/google-cloud-java/pull/1859/commits/4ca6737c1afcf8d80f5afc2816911c136b08d067). 

The reason why I had to comment it is that I need installed artifacts first to be able to pack the fat war file for deployment on GAE (which includes pretty much everything from google-cloud-java, including tests). Integration tests take too much time and fail sporadically, so the easiest way was to turn them off. Since maven is built on convention-over-configuration paradigm, I could not find a way to do install but skip integration-test step (so the only way was to modify the step itself, so it doesn't actually run anything).